### DISCUSSION TITLE
Multiple Jobs Queue

### DISCUSSION THREAD

**Discussion by KSoftUK** [Courseplay/Courseplay_FS22#3357] [Ideas]
For a possible new feature if it hasnt been asked before, did search but didnt find anything:-

adding a queue for jobs if you have the multiple of the same job type such as fertilizing, cultivating, etc. This would work in conjunction with AD (AutoDrive).
I.E
- Add job 1, generate CP, set AD destination
- Add job 2, generate CP, set AD destination
- Add job 2, generate CP, set AD destination
- etc
- Hit Start Job
- Will drive to job 1
- once the helper has finished with one field it would then link to AD to drive to the waypoint and start the next job, seeing as AD can link with CP

**Comment by Tensuko** [comment]
That was asked many times and no we will not implement that.
One reason is that we have enough other things to do, the other is that we never wanted to have a full automation.
